src/App.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './App.css';
+import resumeJson from './assets/resume.json';
+
+import Experience from "./components/Experience/Experience";
+import Study from "./components/Study/Study";
+import Award from "./components/Award/Award";
+import Skill from "./components/Skill/Skill";
+import Language from "./components/Language/Language";
+import Interest from "./components/Interest/Interest";
+import Reference from "./components/Reference/Reference";
+
+function App() {
+  const resumeData = resumeJson;
+  return (
+    <main className="App">
+      <header>
+        <div>
+          <h1>{resumeData.basics.name}</h1>
+          <span className="label">{resumeData.basics.label}</span>
+        </div>
+        <img src={resumeData.basics.picture} />
```

Comment:
Make sure you add ALT text in the future for accessibility purposes. More often than not, it can be the title / description of something from the same object, if you're adding dynamic content.